src/components/wishlist/wishlist.jsx

```diff
@@ -1,42 +1,48 @@
 import React from 'react'
-import Header from "../../assets/camera.jpg"
 import { FaHeart } from "react-icons/fa";
+import { useWishlist } from '../../contexts/wishlist-context';
 
-const Wishlist = () => {
+
+const Wishlist = ({item}) => {
+  const{wishlist,setWishlist}=useWishlist();
+  function removeHandler(item){
+    setWishlist(wishlist.filter((data)=>data._id!==item._id))
+  }
+  
   return (
     <>
-    
     <div className="card-wrapper flex-col">
       <div className="center">
         <div className="image-conatiner">
           <img
             className="card-image"
-            src={Header}
+            src={item.img}
             alt="card-image"
           />
         </div>
          <div class="flex-row space-between">
         
-        <div className="card-header-one">Camera</div>
+        <div className="card-header-one">{item.name}</div>
         <div class="rating-comp flex-row display-flex-center">
-        <span class="star-icon">4
+        <span class="star-icon">{item.rating}
         <FaHeart/>
```

Comment:
you can destructure item  to get its properties

Reply:
okay...will do it